tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       hostPID: true
       containers:
         - name: kube-kata
-          image: quay.io/kata-containers/kata-deploy:latest
+          image: quay.io/kata-containers/kata-deploy-ci:kata-containers-latest
```

Comment:
This change doesn't seem to be related to the PR or I'm missing something ?

Reply:
I second this one!

Reply:
(good old `git commit -as` without reviewing the content...)